src/builder.rs

```diff
@@ -528,12 +613,18 @@ impl BundleMetadata {
     }
 }
 
-/// A builder that constructs a [`Bundle`] from a set of notes to be spent, and outputs
-/// to receive funds.
+/// A tuple containing an in-progress bundle with no proofs or signatures, and its associated metadata.
+#[cfg(feature = "circuit")]
+pub type UnauthorizedBundleWithMetadata<V, FL> = (UnauthorizedBundle<V, FL>, BundleMetadata);
+
+/// A builder for constructing an Orchard [`Bundle`] by specifying notes to spend, outputs to
+/// receive, and assets to burn.
+/// This builder provides a structured way to incrementally assemble the components of a bundle.
```

Comment:
_From str4d ([r2602421894](https://github.com/zcash/orchard/pull/471#discussion_r2602421894))_

```suggestion
///
/// This builder provides a structured way to incrementally assemble the components of a bundle.
```

Reply:
FIXED